app/views/container_topology/show.html.haml

```diff
@@ -0,0 +1,122 @@
+:css
+  * {
+    box-sizing: border-box;
+    font-family: sans-serif;
+    font-size: 15px;
+  }
+  body {
+    margin: 0px;
+  }
+  kubernetes-topology-graph {
+    border: 10px solid lightgray;
+    position: fixed;
+    height: 76%;
+    width: 98%;
+    padding: 10px;
+  }
+  .legend {
+    display: inline-block;
+    padding: 0 10px;
+    font-family: "FontAwesome";
+
+  }
+  .legend label {
+    font-weight: 400;
+  }
+
+  .refresh {
+    display: inline-block;
+    width:35px;
+    height:35px;
+    padding:0;
+  }
+
+  #selected {
+    float: right;
+    display: block;
+    margin-top: 15px;
+  }
+
+  kubernetes-topology-icon {
+    padding: 7px 6px 12px 21px !important;
+  }
+
+  .kube-topology line.ServiceContainerGroup {
+    stroke-linecap: round;
+    stroke-dasharray: 5.5;
+  }
+
+  kubernetes-topology-icon svg {
+    width: 39px !important;;
+    height: 39px !important;;
+    display: block;
+  }
```

Comment:
@abonas The point is, there is no scoping style, you're overriding CSS for the whole application, not just your part.